app/src/main/java/org/standardnotes/notes/LoginActivity.kt

```diff
@@ -57,7 +57,7 @@ class LoginActivity : AppCompatActivity() {
                 hideProgress()
             }
 
-            override fun onFailure(call: Call<SigninResponse>, t: Throwable) {
+            override fun onFailure(call: Call<SigninResponse>?, t: Throwable?) {
```

Comment:
The reason is because it can be called null here: https://github.com/standardnotes/android/pull/126/files/6fb5432d340b7d3ab1627f69c42b92b59739c6b6#diff-dee07c4da1d09f253b0c49355a8e87d5R160